app/Http/Controllers/CampaignsController.php

```diff
@@ -49,6 +49,15 @@ public function index(Request $request)
             }
         }
 
+        // Apply scope for the has_website filter:
+        if (isset($filters['has_website'])) {
+            if (filter_var($filters['has_website'], FILTER_VALIDATE_BOOLEAN)) {
+                $query->whereHasWebsite();
+            } else {
+                $query->whereDoesntHaveWebsite();
```

Comment:
Minor, but what do you think about renaming as `whereDoesNotHaveWebsite` ? I think it's a pinch more readable but not a biggie :neckbeard:

Reply:
That's a good call! Updated in https://github.com/DoSomething/rogue/pull/988/commits/c9672bf202399761a4e955099741f3f9bc37a483